src/transformers/modeling_gpt2.py

```diff
@@ -693,6 +778,20 @@ def __init__(self, config):
 
         self.init_weights()
 
+        self.model_parallel = False
+
+    def parallelize(self, device_map):
```

Comment:
Maybe write the docstrings once and for all in a global constant and use our decorators to add it there too?

Reply:
Yes, excellent suggestion. Implemented.